src/frontend/src/styles/main.css

```diff
@@ -405,5 +394,120 @@ hr {
   background-color: rgba(243, 244, 246, 1);
   max-width: 42rem;
   height: 100%;
-  position: fixed;
+}
```

Comment:
Here I removed `position: fixed`, which I should never have merged to start with. The content grows beyond the parent element now on the FAQ and About page, but it's much better that not being able to scroll. Will need to fix this separately.